common/djangoapps/enrollment/views.py

```diff
@@ -71,7 +71,12 @@ class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     rate = '40/minute'
 
     def allow_request(self, request, view):
-        return self.has_api_key_permissions(request) or super(EnrollmentUserThrottle, self).allow_request(request, view)
+        user = request.user
+
+        if user.is_superuser or user.is_staff:
```

Comment:
We increase it again. Removing the rate altogether introduces additional risk if a staff member's account is compromised. If someone manages to get one of my API tokens, that person can access a lot of data. A throttle helps ensure we don't lose too much data before locking things down.